### PR TITLE
[UBS] - update the data in the bag table #5204

### PR DESCRIPTION
**Pull request by juseti**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

## Link to issue

https://github.com/ita-social-projects/GreenCity/issues/5204

## Summary of change

DatabaseChangeLog created and created_by_id, created_at, description and description_eng data entered to the bag table.

## Testing approach

Chacked data in the columns created_by_id, created_at, description and description_eng in the bag table.

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: dao/src/main/resources/db/changelog/logs/ch-update-bag-created-at-created-by-id-description-description-eng-Seti.xml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet id="Seti-5" author="Julia Seti">
+        <update tableName="bag">
+            <column name="created_at" value="2023-01-26"/>
```
Comment (RomaMocherniuk) [inline dao/src/main/resources/db/changelog/logs/ch-update-bag-created-at-created-by-id-description-description-eng-Seti.xml]: today is `2023-02-07`

File: dao/src/main/resources/db/changelog/logs/ch-update-bag-created-at-created-by-id-description-description-eng-Seti.xml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+            <column name="created_by_id" value="1"/>
+            <column name="description" value="Тариф на послугу 'Текстильні відходи' - 20"/>
+            <column name="description_eng" value="Tariff for Service 'Textile waste' - 20"/>
+            <where>id=5</where>
+        </update>
+
+        <update tableName="bag">
+            <column name="created_at" value="2023-01-26"/>
+            <column name="created_by_id" value="1"/>
+            <column name="description" value="Тариф на послугу 'Текстильні відходи' - 60"/>
```
Comment (RomaMocherniuk) [inline dao/src/main/resources/db/changelog/logs/ch-update-bag-created-at-created-by-id-description-description-eng-Seti.xml]: Why do you add  it twice`Тариф на послугу 'Текстильні відходи' - 60`?

File: dao/src/main/resources/db/changelog/logs/ch-update-bag-created-at-created-by-id-description-description-eng-Seti.xml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+            <column name="created_by_id" value="1"/>
+            <column name="description" value="Тариф на послугу 'Мікс відходів' - 120"/>
+            <column name="description_eng" value="Tariff for Service 'Mix waste' - 120"/>
+            <where>id=4</where>
+        </update>
+
+        <update tableName="bag">
+            <column name="created_at" value="2023-01-26"/>
+            <column name="created_by_id" value="1"/>
+            <column name="description" value="Тариф на послугу 'Текстильні відходи' - 20"/>
```
Comment (RomaMocherniuk) [inline dao/src/main/resources/db/changelog/logs/ch-update-bag-created-at-created-by-id-description-description-eng-Seti.xml]: same here

File: dao/src/main/resources/db/changelog/logs/ch-update-bag-created-at-created-by-id-description-description-eng-Seti.xml [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+            <column name="created_by_id" value="1"/>
+            <column name="description" value="Тариф на послугу 'Текстильні відходи' - 60"/>
+            <column name="description_eng" value="Tariff for Service 'Textile waste' - 60"/>
+            <where>id=3</where>
+        </update>
+
+        <update tableName="bag">
+            <column name="created_at" value="2023-01-26"/>
+            <column name="created_by_id" value="1"/>
+            <column name="description" value="Тариф на послугу 'Мікс відходів' - 120"/>
```
Comment (RomaMocherniuk) [inline dao/src/main/resources/db/changelog/logs/ch-update-bag-created-at-created-by-id-description-description-eng-Seti.xml]: and here

File: dao/src/main/resources/db/changelog/logs/ch-update-bag-created-at-created-by-id-description-description-eng-Seti.xml (hunk 2 above)
Comment (juseti) [reply dao/src/main/resources/db/changelog/logs/ch-update-bag-created-at-created-by-id-description-description-eng-Seti.xml]: services for different tariffs have the same names

File: dao/src/main/resources/db/changelog/logs/ch-update-bag-created-at-created-by-id-description-description-eng-Seti.xml (hunk 1 above)
Comment (juseti) [reply dao/src/main/resources/db/changelog/logs/ch-update-bag-created-at-created-by-id-description-description-eng-Seti.xml]: fixed

File: dao/src/main/resources/db/changelog/logs/ch-update-bag-created-at-created-by-id-description-description-eng-Seti.xml (hunk 2 above)
Comment (juseti) [reply dao/src/main/resources/db/changelog/logs/ch-update-bag-created-at-created-by-id-description-description-eng-Seti.xml]: description changed

**Review by JJJazl — APPROVED** [review]

**Review by NKorzhik — APPROVED** [review]

**Review by LiliaMokhnatska — APPROVED** [review]

**Review by RomaMocherniuk — APPROVED** [review]

File: dao/src/main/resources/db/changelog/logs/ch-update-bag-created-at-created-by-id-description-description-eng-Seti.xml [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+            <column name="description" value="Послуга 'Текстильні відходи' - 60 за тарифом 1"/>
+            <column name="description_eng" value="Service 'Textile waste' - 60 by tariff 1"/>
+            <where>id=3</where>
+        </update>
+
+        <update tableName="bag">
+            <column name="created_at" value="2023-02-07"/>
+            <column name="created_by_id" value="1"/>
+            <column name="description" value="Послуга 'Мікс відходів' - 120 за тарифом 2"/>
+            <column name="description_eng" value="TService 'Mix waste' - 120 by tariff 2"/>
```
Comment (RomaMocherniuk) [inline dao/src/main/resources/db/changelog/logs/ch-update-bag-created-at-created-by-id-description-description-eng-Seti.xml]: `TService`?

**Pull request merged**